### DISCUSSION TITLE
Container-specific Essentials

### DISCUSSION THREAD

**Discussion by mjtys** [zen-browser/desktop#2815] [Ideas]
Hi all,

I'd love for the newly added global essentials to be container specific, similarly to how Arc treats favourites. 

Arc has profiles/spaces whereas Zen has containers/workspaces. Arc syncs favourites across profiles and not globally. 

Example being if I have a personal container and a work container, I don't want my personal essentials visible with my work essentials, however, if I have two work workspaces I'd like my work essentials to be visible on both workspaces and the same with personal. 

I hope this makes sense and keep up the great work! 👏🏻

**Comment by DaveyLovesCode** [comment]
Definitely defeats the purpose of workspaces for me. There seems to be a lot of support for adding a setting to let the user decide which way they like it:
https://www.reddit.com/r/zen_browser/comments/1gnrobm/i_love_the_new_direction_with_essentials_but/

**Comment by Arilas** [comment]
Plus one. I have 4 containers, each with their own Gmail and Calendar. It was organized in a way, that for each Workspace I have Gmail and Calendar in the same place, and I don't need to search for them in a list of tabs.

So right now I cannot place them in Essentials, only in a Pinned tabs list, which is not so easy to navigate with a lot of tabs

**Reply by nijin26** [reply]
I am also facing the same issue. It was good till the a17 version, this latest update making it a mess. They should provide an option to show the pinned tabs as icons instead of normal tabs ([Superpins](https://zen-browser.app/mods/ad97bb70-0066-4e42-9b5f-173a5e42c6fc) mod do the same, I am currently using it for now). Essentials doesn't make any sense to me (in my workflow), i don't want any tabs globally accessible in all of my workspace. 

I have each workspace for there own purpose, so each of them have there own pinned tabs (it might be more than 5), then showing it as list is messy and hard to handle.

**Comment by elliotdickison** [comment]
I just switched from Arc to Zen and I have to say that the global essentials has been a _huge_ win for me. Like probably one of my top 3 features of Zen so far 😅. I use spaces heavily, but I have a few tabs that I want available from _everywhere_. In Arc I was constantly losing these tabs, having sessions in them get out of sync between spaces, etc. In Zen I've got my, well, _essential_ tabs at the top and they are always there! The SuperPins mod makes my pinned tabs look like mini essentials specific to each workspace (not for everyone I'm sure but fits my needs perfectly).

**Reply by 12th-devs** [reply]
Oh dev added this as a setting (container specific essentials) awhile ago 😆

**Comment by niderhoff** [comment]
Zen Settings -> Tab Management -> Pinned Tabs -> Enable container-specific essentials (bottom option)

**Reply by SaintFrans** [reply]
YOU ABSOLUTE LEGEND! I am still wondering why this isn't the default option when creating multiple containers, but this made me switch to zen browser all the way. This was my only downfall with the browser.
